editor/src/clj/editor/pipeline/texture_set_gen.clj

```diff
@@ -30,11 +34,6 @@
     (TextureSetGenerator$AnimDesc. (:id anim) (protobuf/val->pb-enum Tile$Playback (:playback anim)) (:fps anim)
                                    (:flip-horizontal anim) (:flip-vertical anim))))
 
-(defn- map->Rect
-  [{:keys [path width height]}]
-  ;; NOTE: The other attributes do not matter for our use case.
-  (TextureSetLayout$Rect. path -1 (int width) (int height)))
-
```

Comment:
Removed and inlined this at the call sites, since it is specialized to particular use cases.